lisa/tools/azsecd.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from lisa.executable import Tool
```

Comment:
Because it's an Azure specific tool, please put it in lisa.sut_orchestrator.azure.tools. Refer to `KvpClient`